build/roslib.js

```diff
@@ -1579,7 +1600,7 @@ TFClient.prototype.processTFArray = function(tf) {
         cb(info.transform);
       });
     }
-  });
+  }, this);
```

Comment:
I prefer using 'that' instead of passing 'this' to forEach. It's harder to make a mistake that way if you have nested functions or async calls.